sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -235,7 +235,7 @@ case class AlterTableSetPropertiesCommand(
     // direct property.
     val newTable = table.copy(
       properties = table.properties ++ properties,
-      comment = properties.get("comment"))
+      comment = properties.get("comment").orElse(table.comment))
```

Comment:
alter table src set tblproperties ('foo' = 'bar', 'comment' = 'table_comment');
alter table src unset tblproperties ('foo');
we will lost comment in this case.